.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,36 @@
+# Issue summary
+
+Write a short description of the issue here ↓
+
+
+
+## Expected behavior
+
+What do you think should happen?
+
+
+
+## Actual behavior
+
+What actually happens?
+
+Tip: include an error message (in a `<details></details>` tag) if your issue is related to an error
+
+
+
+## Steps to reproduce the problem
+
+1.
+1.
+1.
+
+## Logs
+Enable logs with `export SHOPIFY_LOG=1` and paste them here from "shopify_api.log"
```

Comment:
```suggestion
Enable logs with `export SHOPIFY_LOG=1` and paste them here from "shopify_api.log"
```
If applicable, enable the logs as described in the README, and paste the relevant portion here.